### PR TITLE
Lesson3

### PR DESCRIPTION
**Pull request by gittafia**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,150 @@
...
+            id="comment"
+            cols="30"
+            rows="2"
+            placeholder="Comment"
+            maxlength="500"
+            ></textarea>
+            <br/>
+            <label for="answer">What does HTML stand for?</label>
+            <br/>
+            <select name="" id="">
```
Comment (ghost) [inline index.html]: The name value is required.

File: index.html [hunk 2]
```diff
@@ -9,5 +9,150 @@
...
+    </table>
+    <br/>
+    <br/>
+    <br/>
+    <form action="/" method="GET">
+        <fieldset>
+            <legend>My first form</legend>
+            <label for="name">Name</label>
+            <br/>
+            <input type="text" id="name" name="name" placeholder="Name"/>
```
Comment (ghost) [inline index.html]: Should be required.

File: index.html (hunk 1 above)
Comment (gittafia) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (gittafia) [reply index.html]: Fixed.

**Pull request merged**